packages/noya-state/src/selectors/pointSelectors.ts

```diff
@@ -233,6 +240,14 @@ export const moveSelectedPoints = (
         y: getNewValue(layer.frame.y, mode, delta.y),
       };
 
+      if (draftLayer.frame.height === 0) {
+        draftLayer.frame.height = 1;
```

Comment:
When we adjust the layer like this, do we need to move the points to `y = 0.5`? Otherwise they'll presumably be at the top or bottom of the frame, which isn't quite right